.circleci/config.yml

```diff
@@ -25,23 +19,29 @@ jobs:
     working_directory: ~/repo
     docker:
       - image: cimg/go:1.19
+    steps: *simple_job_steps
+
+  build-1-20:
+    working_directory: ~/repo
+    docker:
+      - image: cimg/go:1.20
     steps:
       - checkout
       - run:
           name: Run tests and linters
           command: |
             make ci
 
-  build-1-20:
+  build-1-21:
     working_directory: ~/repo
     docker:
-      - image: cimg/go:1.20
+      - image: cimg/go:1.21
     steps: *simple_job_steps
 
 workflows:
   pr-build-test:
     jobs:
-      - build-1-17
```

Comment:
Do we already make it explicit which Go versions we test against?

Reply:
We can't actually build against 1.17 anymore, there's a dep in the chain that requires 1.18 so I just bumped it up. 🤷